index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Walter's Tic Tac Toe</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <script src="script.js"></script>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  </head>
+  <body>
+    <div class="top">
+      <div id="topLeft" onclick="play();"></div>
```

Comment:
It's best practice not to use `onclick` handlers in HTML to setup events.  It's better to use `addEventListener` in your JS. 

This is to maintain separation of concerns. HTML should only concern structure and content, not functionality (that should go in JS).